hyperoptimization/run_best_config.py

```diff
@@ -0,0 +1,72 @@
+import hpbandster.core.result as hpres
+from hyperoptimization.runs import from_config as run
+from hyperoptimization.workers import TVAEWorker
```

Comment:
Do I understand correctly that this method is tailored to a particular model?

Reply:
Models with different hyperparameters need to have them declared. However the method differentiates bewteen models and variational optimization schemes.